README.md

```diff
@@ -90,6 +90,23 @@ conda install scikit-learn-intelex -c intel
 
 </details>
 
+⚠️ Note: *dpcpp support is optional dependency (required dependencies for support dpcpp
```

Comment:
There is no information why the user needs to install this dependency.

Reply:
that's better?